Makefile

```diff
@@ -99,6 +99,7 @@ $(BUILDDIR)/%/:
 .PHONY: build
 build: $(BUILDDIR)/bin/  ## build the binary from source
 	go build -ldflags="-X main.version=${VERSION}" -o $<image-builder ./cmd/image-builder/
+	(cd ./cmd/image-builder && ln -sf image-builder bootc-image-builder)
```

Comment:
@lzap you mean this symlink here in your comment in https://github.com/osbuild/image-builder-cli/pull/374#issuecomment-3569787371 ? If so the idea behind this one was that it would be nice to provide the bib binary everywhere where we have ibcli so that its easier for users to switch - *but* thinking about this again maybe its not such a good ideas as we want people to move to bib and by making it too easy to just stick with bib we defeat this goal. Idk, happy to remove this again and reconsider.

Reply:
Yes, I meant this link.

Don't we only support bib via its container? Then introducing this link is actually a step back if we distribute it with CLI RPM and we might accidentally acquire more users of bib who were previously not exposed. That is why I thought just building differently named binary would be enough. In fact, if bib is only consumed via podman, then we could just introduce an environment variable because the name of the binary does not matter as it is never directly used it is `docker` or `podman` what is typed in.